lib/stdio/utils.js

```diff
@@ -7,3 +8,16 @@ export const isBinary = value => isUint8Array(value) || Buffer.isBuffer(value);
 
 const textDecoder = new TextDecoder();
 export const binaryToString = uint8ArrayOrBuffer => textDecoder.decode(uint8ArrayOrBuffer);
+
+// Like `source.pipe(destination)`, if `source` ends, `destination` ends.
+// Like `stream.pipeline(source, destination)`, if `source` aborts/errors, `destination` aborts.
+// Unlike `stream.pipeline(source, destination)`, `destination` behavior does not propagate to `source`.
```

Comment:
It's unclear what exactly "behavior" means here.

Reply:
Fixed in e8f3cab